config/samples/k6_v1alpha1_k6_with_localfile.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: k6.io/v1alpha1
+kind: K6
+metadata:
+  name: k6-sample
+spec:
+  parallelism: 4
+  script:
+    localFile: /test/test.js
```

Comment:
Please correct me if I'm wrong but this example makes sense when there is also a runner image defined right? So for completeness, the example should also have this part:
```
  runner:
      image: <custom-image>
```